### DISCUSSION TITLE
Automatically Notify Organization Users of Long Running Actions that fall outside of average max quantity

### DISCUSSION THREAD

**Discussion by OffensiveBias-08-145** [community/community#18162] [Actions]
When actions are running, there is currently not a system in place (that I am aware of) to notify organizational owners of long running actions that fall outside of normal operating quantity.

For example, our average Actions run roughly 10-20 min for some of our test suites. We had actions that were running in excess of 100+ min (Max was 963 min) due too some unknown issue during runtime. Since we received no notice that this action was still running, it ate through our 3k included actions min. 

While there are means to limit the amount of time an action may run, this is not developer friendly as it requires continuous monitoring of action running times across a workflow to account for increased runtimes as the test suite grows.

**Comment by emilqnn** [comment]
днеска сутринта ли
